File: api/DTOs/Admin/EmploymentDto.cs
namespace api.DTOs.Admin
{
    public class EmploymentDto
    {
        public int SelectionDecisionId { get; set; }    
        public int CvRefId { get; set; }
        public int ApplicationNo { get; set; }
        public string CandidateName {get; set;}
        public string ProfessionName {get; set;}
        public string CompanyName { get; set; }
        public string SelectionStatus { get; set; }
        public DateTime SelectedOn { get; set; }
        public int ChargesFixed {get; set;}
        public int Charges {get; set;}
        public string SalaryCurrency { get; set; }
        public int Salary { get; set; }
        public int ContractPeriodInMonths { get; set; }=24;
        public int WeeklyHours {get; set;}=48;
        public bool HousingProvidedFree { get; set; }
        public bool HousingNotProvided {get; set;}
        public int HousingAllowance { get; set; }
        public bool FoodProvidedFree { get; set; }
        public int FoodAllowance { get; set; }
        public bool FoodNotProvided { get; set; }
        public bool TransportProvidedFree { get; set; }
        public bool TransportNotProvided { get; set; }
        public int TransportAllowance { get; set; }
        public int OtherAllowance { get; set; }
        public int LeavePerYearInDays { get; set; }
        public int LeaveAirfareEntitlementAfterMonths {get; set;}
        public string OfferAccepted { get; set; }
        public DateTime OfferAcceptedOn { get; set; }
       
    }
}

File: client/src/app/_dtos/admin/employmentDto.ts
export interface IEmploymentDto
{
     applicationNo: number;
     candidateName: string;
     companyName: string;
     categoryRef: string;

     selectionDecisionId: number;
     cvRefId: number;
     selectedOn: Date;
     charges: number;
     weeklyHours: number;
     salaryCurrency: string;
     salary: number;
     contractPeriodInMonths: number;
     housingProvidedFree: boolean;
     housingAllowance: number;
     housingNotProvided: boolean;
     foodProvidedFree: boolean;
     foodNotProvided: boolean;
     foodAllowance: number;
     transportProvidedFree: boolean;
     transportAllowance: number;
     transportNotProvided: boolean;
     otherAllowance: number;
     leavePerYearInDays: number;
     leaveAirfareEntitlementAfterMonths: number;
     offerAccepted: string;
     offerAcceptedOn: Date;
     remarks: string;
}
